wakuv2/api_test.go

```diff
@@ -23,6 +23,8 @@ import (
 	"testing"
 	"time"
 
+	"github.com/waku-org/go-waku/waku/v2/protocol/relay"
+
 	"github.com/status-im/status-go/wakuv2/common"
```

Comment:
Not a point for this PR, but I wonder if we should consider the `waku-org` organisation repos as 1st party packages in the same way we treat `ethereum` org packages. Not important, but just a thought.

Reply:
Please expand on this idea :thinking:  ,   I did not know that we had an special treatment for ethereum-org packages except for a go-ethereum fork

Reply:
Ah this is nothing fancy. Our linter is configure to enforce the grouping of packages in the following way:

- `std lib`
- 3rd party packages
- `go-ethereum` and `status-go` packages

To me `waku-org` packages are not strictly 3rd party although clearly not "1st party". But if we treat `ethereum` org packages as something special, your work in `waku-org` really should have the same treatment as a kind of integral package. The rules could be:

- `std lib`
- 3rd party packages
- `go-ethereum`, `status-go` and `waku-org` packages

Reply:
Oh I see! good idea!